src/mlpack/tests/data/train_data_5_classes.csv

```diff
@@ -0,0 +1,63 @@
+4.7,3.2,1.6,0.2,0
```

Comment:
If this file isn't used, can you remove it from the repository please?

Reply:
Done